src/logo/builtin.c

```diff
@@ -775,6 +775,26 @@ static const FFlogo* getLogoKDENeon()
     FF_LOGO_RETURN
 }
 
+static const FFlogo* getLogoKISSLinux()
+{
+    FF_LOGO_INIT
+    FF_LOGO_NAMES("kiss", "kiss-linux", "kisslinux")
+    FF_LOGO_LINES(
+        "    ___      \n"
+        "   (.· |     \n"
+        "   (<> |     \n"
+        "  / __  \\    \n"
+        " ( /  \\ /|   \n"
+        "_/\\ __)/_)   \n"
+        "\\/-____\\/    \n"
```

Comment:
The logo is solely magenta, add $2 to change to blue and $1 to change back to magenta. The neofetch logo also has some white in it.

Reply:
Fixed